core/src/main/java/org/elasticsearch/index/query/MatchQueryBuilder.java

```diff
@@ -44,8 +44,9 @@
  * can construct different queries based on the type provided.
  */
 public class MatchQueryBuilder extends AbstractQueryBuilder<MatchQueryBuilder> {
-    public static final ParseField MATCH_PHRASE_FIELD = new ParseField("match_phrase", "text_phrase");
-    public static final ParseField MATCH_PHRASE_PREFIX_FIELD = new ParseField("match_phrase_prefix", "text_phrase_prefix");
+    public static final ParseField MATCH_PHRASE_FIELD = new ParseField("match_phrase", "text_phrase").withAllDeprecated("match");
+    public static final ParseField MATCH_PHRASE_PREFIX_FIELD = new ParseField("match_phrase_prefix", "text_phrase_prefix")
+            .withAllDeprecated("match");
```

Comment:
This was incorrectly added. You can add `ParseField#withAllDeprecated(String)` if the entire field is now deprecated and replaced with some other thing. It will then flag all forms of the field as deprecated with a message and the String identifies what the field is replaced with. In this case we don't actually need it as we haven't replaced `match_phrase` and `match_phrase_prefix` we have just changed how they are parsed internally.